code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -49,8 +54,30 @@
 
 	var/obj/item/radio/integrated/signal/sradio // AI's signaller
 
+	var/holoform = FALSE
+	var/canholo = TRUE
+	var/obj/item/weapon/card/id/access_card = null
+	var/chassis = "repairbot"
+	var/list/possible_chassis = list("cat", "mouse", "monkey", "corgi", "fox", "repairbot")
+
+	var/emitterhealth = 50
+	var/emittermaxhealth = 50
+	var/emitterregen = 1
+	var/emittercd = 10
+	var/emittersemicd = FALSE
+
+	var/overload_ventcrawl = 0
+	var/overload_bulletblock = 0	//Why is this a good idea?
+	var/overload_maxhealth = 0
+	canmove = FALSE
+
+/mob/living/silicon/pai/Destroy()
+	pai_list -= src
+	..()
 
 /mob/living/silicon/pai/New(var/obj/item/device/paicard/P)
+	START_PROCESSING(SSfastprocess, src)

```

Comment:
Future features might use it too, like doorjack's progress bars which I'l be touching up on later..?